man/systemd-cryptsetup-generator.xml

```diff
@@ -165,6 +181,38 @@
         the initrd.</para>
         </listitem>
       </varlistentry>
+
+      <varlistentry>
+        <term><varname>luks.data_dev=</varname></term>
+        <term><varname>rd.luks.data_dev=</varname></term>
```

Comment:
hmm, can you reorder this man page so that the parameters `luks.name=`, `luks.data_dev=`, `luks.key=`, `luks.options=` appear in the same order as the columns in `/etc/crypttab` please? 

Maybe even add a reference to each of the four parameters, that mention this relationship, i.e. for each of the params say that "This option corresponds to the first/second/third/fourth column of the crypttab(5) file.".

Hmm, given that the option for the header is just called "header=", and not "header_dev=", and the key spec also is just called "key=" instead of "key_dev=" or "key_file=" I think we should also name the kernel cmdline option for the data device as "luks.data". I think that would be less surprising as a name, following the scheme otherwise use for naming.